### ISSUE TITLE
pyproj 2.6.0:    ProjError: proj error: latitude or longitude exceeded limits 

### ISSUE THREAD

**Issue by evanburgess** [pyproj4/pyproj#719]
This may be similar to an issue raised for a previous version [here](https://github.com/pyproj4/pyproj/issues/192) but I am running `pyproj                    2.6.0            py38h95a477d_0    conda-forge`

```
from pyproj.proj import Proj
p = Proj('EPSG:32611'). # this is utm z11N
```

If I provide `get_factors` two coordinates within Zone11, in order longitude, latitude as stated in the args, I get an error:

```
In [73]: p.get_factors(-119.14491266, 38.05781173,errcheck=True)
---------------------------------------------------------------------------
ProjError                                 Traceback (most recent call last)
<ipython-input-73-b2195b5241d9> in <module>
----> 1 p.get_factors(-119.14491266, 38.05781173,errcheck=True)

~/anaconda3/envs/run/lib/python3.8/site-packages/pyproj/proj.py in get_factors(self, longitude, latitude, radians, errcheck)
    224
    225         # calculate the factors
--> 226         factors = self._get_factors(inx, iny, radians=radians, errcheck=errcheck)
    227
    228         # if inputs were lists, tuples or floats, convert back.

pyproj/_proj.pyx in pyproj._proj._Proj._get_factors()

ProjError: proj error: latitude or longitude exceeded limits
```

Interestingly if I swap the longitude and latitude the error goes away.
```
In [68]: p.get_factors(38.05781173, -119.14491266, errcheck=True)
Out[68]: Factors(meridional_scale=1.0599574025919634, parallel_scale=1.0599574025615974, areal_scale=1.123509695277315, angular_distortion=0.0, meridian_parallel_angle=90.0, meridian_convergence=163.99444980916587, tissot_semimajor=1.0599574025767804, tissot_semiminor=1.0599574025767804, dx_dlam=-0.8032681213937519, dx_dphi=-0.29141737285659497, dy_dlam=0.23041763939080082, dy_dphi=-1.0159217245120367)
```
However, if I remove the negative on the longitude it doesn't change things.
```
In [66]: p.get_factors(38.05781173, 119.14491266, errcheck=True)
Out[66]: Factors(meridional_scale=1.0599574025919634, parallel_scale=1.0599574025615974, areal_scale=1.123509695277315, angular_distortion=0.0, meridian_parallel_angle=90.0, meridian_convergence=163.99444980916587, tissot_semimajor=1.0599574025767804, tissot_semiminor=1.0599574025767804, dx_dlam=-0.8032681213937519, dx_dphi=-0.29141737285659497, dy_dlam=0.23041763939080082, dy_dphi=-1.0159217245120367)
```

**Comment by snowman2** [comment]
Can you show the output from: `python -c "import pyproj; pyproj.show_versions()"`?

**Comment by snowman2** [comment]
```
>>> import pyproj
>>> p = pyproj.Proj('EPSG:32611')
>>> p.get_factors(-119.14491266, 38.05781173, errcheck=True)
Factors(meridional_scale=1.0000361534159121, parallel_scale=1.0000361533537612, areal_scale=1.0000723080767404, angular_distortion=0.0, meridian_parallel_angle=89.99999879258174, meridian_convergence=-1.3226327138341294, tissot_semimajor=1.0000361533848365, tissot_semiminor=1.0000361533848365, dx_dlam=0.7882110595745957, dx_dphi=0.023016327063349325, dy_dlam=-0.018198529813662212, dy_dphi=0.9968785236708076)
>>> p.get_factors(119.14491266, 38.05781173, errcheck=True)
Factors(meridional_scale=1.3213493178859577, parallel_scale=1.3213493179551117, areal_scale=1.7459640199690618, angular_distortion=1.2922754561151397e-06, meridian_parallel_angle=89.99999879258174, meridian_convergence=-137.32996862897863, tissot_semimajor=1.3213493328216959, tissot_semiminor=1.3213493030193735, dx_dlam=-0.7659608476207057, dx_dphi=0.892987889070107, dy_dlam=-0.7060669012903807, dy_dphi=-0.9687378900280484)
>>> pyproj.show_versions()
pyproj info:
    pyproj: 2.6.1.post1
      PROJ: 7.0.1
  data dir: ~/venv/lib/python3.8/site-packages/pyproj/proj_dir/share/proj

System:
    python: 3.8.5 (default, Jul 28 2020, 12:59:40)  [GCC 9.3.0]
executable: ~/venv/bin/python
   machine: Linux-5.4.0-48-generic-x86_64-with-glibc2.29

Python deps:
       pip: 20.0.2
setuptools: 44.0.0
    Cython: None

```

**Comment by evanburgess** [comment]
```
>>> import pyproj
>>> p = pyproj.Proj('EPSG:32611')
>>> p.get_factors(-119.14491266, 38.05781173, errcheck=True)
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/ubuntu/anaconda3/envs/run/lib/python3.8/site-packages/pyproj/proj.py", line 226, in get_factors
    factors = self._get_factors(inx, iny, radians=radians, errcheck=errcheck)
  File "pyproj/_proj.pyx", line 327, in pyproj._proj._Proj._get_factors
pyproj.exceptions.ProjError: proj error: latitude or longitude exceeded limits
>>> pyproj.show_versions()
pyproj info:
    pyproj: 2.6.0
      PROJ: 6.3.1
  data dir: /home/ubuntu/anaconda3/envs/run/share/proj

System:
    python: 3.8.2 | packaged by conda-forge | (default, Apr 16 2020, 18:04:51)  [GCC 7.3.0]
executable: /home/ubuntu/anaconda3/envs/run/bin/python
   machine: Linux-5.3.0-1017-aws-x86_64-with-glibc2.10

Python deps:
       pip: 20.0.2
setuptools: 46.1.3.post20200325
    Cython: None```

To state the obvious this is on anaconda and all conda-forge.

**Comment by snowman2** [comment]
I tested conda and pip and both worked:
```
conda create -n pyproj pyproj
conda activate pyproj
```
```
>>> import pyproj
>>> p = pyproj.Proj('EPSG:32611')
>>> p.get_factors(119.14491266, 38.05781173, errcheck=True)
Factors(meridional_scale=1.3213493178821842, parallel_scale=1.3213493179602884, areal_scale=1.7459640199709172, angular_distortion=0.0, meridian_parallel_angle=90.0, meridian_convergence=-137.32996862915613, tissot_semimajor=1.3213493179212363, tissot_semiminor=1.3213493179212363, dx_dlam=-0.7659608476262568, dx_dphi=0.8929878890645558, dy_dlam=-0.7060669012903807, dy_dphi=-0.9687378900280484)
>>> p.get_factors(-119.14491266, 38.05781173, errcheck=True)
Factors(meridional_scale=1.0000361534159201, parallel_scale=1.0000361533537612, areal_scale=1.0000723080767486, angular_distortion=0.0, meridian_parallel_angle=90.0, meridian_convergence=-1.3226327138540594, tissot_semimajor=1.0000361533848405, tissot_semiminor=1.0000361533848405, dx_dlam=0.7882110595745957, dx_dphi=0.02301632706369627, dy_dlam=-0.018198529813662212, dy_dphi=0.9968785236708076)
>>> pyproj.show_versions()
pyproj info:
    pyproj: 2.6.1.post1
      PROJ: 7.1.1
  data dir: ~/miniconda/envs/pyproj/share/proj

System:
    python: 3.8.6 | packaged by conda-forge | (default, Oct  7 2020, 19:08:05)  [GCC 7.5.0]
executable: ~/miniconda/envs/pyproj/bin/python
   machine: Linux-5.4.0-48-generic-x86_64-with-glibc2.10

Python deps:
       pip: 20.2.3
setuptools: 49.6.0.post20201009
    Cython: None

```
```
cat ~/.condarc
channels:
  - conda-forge
  - defaults
always_yes: true
channel_priority: strict
```

**Comment by snowman2** [comment]
https://pyproj4.github.io/pyproj/stable/history.html. Was addressed in #600. Fix added in 2.6.1.
